### PR TITLE
added clarification for multiple queries

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4013

### CODE REVIEW

File: windows/security/threat-protection/windows-firewall/create-wmi-filters-for-the-gpo.md [hunk 1]
```diff
@@ -89,6 +89,9 @@ First, create the WMI filter and configure it to look for a specified version (o
 
 10. Click **Save** to save your completed filter.
 
+>[!NOTE]
```
Comment (illfated) [inline windows/security/threat-protection/windows-firewall/create-wmi-filters-for-the-gpo.md]: recommended MarkDown codestyle, single space between quote marker and following text:
```suggestion
> [!NOTE]
```

**Comment by e0i** [comment]
@nenonix Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by Dansimp** [comment]
copy edited

**Pull request merged**